src/app/login-button/login-button.component.ts

```diff
@@ -0,0 +1,27 @@
+// Adding back Alexander's work
+import { Component, OnInit } from '@angular/core';
+import {Router} from '@angular/router';
+import { AuthService } from '../core/auth.service';
+
+@Component({
+  selector: 'app-login-button',
+  templateUrl: './login-button.component.html',
+  styleUrls: ['./login-button.component.scss']
+})
+export class LoginButtonComponent implements OnInit {
+
+  constructor(public auth: AuthService, private router: Router) { }
+
+  ngOnInit(): void {
+    const loginButton = document.getElementById('login_button');
+    loginButton.style.display = 'block';
```

Comment:
Why don't you just style the button in the .scss file?

Reply:
Valid point, I was copying and pasting at that point